load-balancing/elb-v2/common_functions.sh

```diff
@@ -374,15 +381,21 @@ wait_for_state() {
         return 1
     fi
 
-    msg "Checking $WAITER_ATTEMPTS times, every $waiter_interval seconds, for instance $instance_id to be in state $state_name"
+    # Check if a custom waiter_attempts was passed into the function
+    # and override the attemps if true
+    if [ -z "$waiter_attempts" ]; then
```

Comment:
I think you should go back to ${var+waiter_attempts} because it distinguishes between unset and empty string: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash